### ISSUE TITLE
Popup Bounded Button Missing Text & is Inaccessible 

### ISSUE THREAD

**Issue by emilyrohrbough** [cerner/terra-core#1353]
# Issue Description
The close button in the [bounded popup](http://engineering.cerner.com/terra-core/#/tests/popup-tests/bounded) is missing text, causing it to be inaccessible. 

Additionally, it appear to be applying the classname 'close' which is provides not additional styling so needs removed.

<img width="607" alt="screen shot 2018-03-16 at 7 24 42 am" src="https://user-images.githubusercontent.com/14099737/37520550-8350402a-28eb-11e8-9c3c-18666d0a19e4.png">

## Issue Type

- [ ] New Feature
- [ ] Enhancement
- [x] Bug
- [ ] Other

## Expected Behavior
Be accessible.

**Comment by emilyrohrbough** [comment]
Moved issue to the terra-framework repository.
